src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public class RightTriangle extends Figure {
+    private int legA;
+    private int legB;
```

Comment:
Let's make variable names more informative. I don't understand what is "leg" of right triangle)